apps/web/src/components/sidebar/IndexingStatus/index.tsx

```diff
@@ -4,16 +4,23 @@ import { getIndexingStatus } from '@safe-global/safe-gateway-typescript-sdk'
 import useAsync from '@/hooks/useAsync'
 import useChainId from '@/hooks/useChainId'
 import ExternalLink from '@/components/common/ExternalLink'
+import useIntervalCounter from '@/hooks/useIntervalCounter'
 
 const STATUS_PAGE = 'https://status.safe.global'
 const MAX_SYNC_DELAY = 1000 * 60 * 5 // 5 minutes
+const POLL_INTERVAL = 1000 * 60 // 1 minute
 
 const useIndexingStatus = () => {
   const chainId = useChainId()
+  const count = useIntervalCounter(POLL_INTERVAL)
```

Comment:
This is causing excessive requests because the hook rerenders in the parent component and is not limited to `useIndexingStatus`

Reply:
For context we ran into this issue before and the solution back then was to not use this hook but this is not a viable option here: https://github.com/safe-global/safe-wallet-monorepo/pull/4003#discussion_r1697155585